docs/specs/0023-gate-onboarding-on-first-launch.md

```
@@ -0,0 +1,98 @@
+# SPEC: feat(onboarding): gate onboarding on first launch
+
+## Problem
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)